arches/management/commands/resources.py

```diff
@@ -44,11 +44,22 @@ def add_arguments(self, parser):
             help="A graphid of the Resource Model you would like to remove all instances from.",
         )
 
+        parser.add_argument(
+            "-e",
+            "--editlog",
+            action="store_true",
+            dest="editlog",
+            help="used to clear the edit log. If a graphid is provided, only the edit log for that graph will be cleared.",
+        )
+
     def handle(self, *args, **options):
         if options["operation"] == "remove_resources":
-            self.remove_resources(force=options["yes"], graphid=options["graph"])
+            self.remove_resources(force=options["yes"], graphid=options["graph"], clear_edit_log=options["editlog"])
+
+        if options["operation"] == "clear_edit_log":
```

Comment:
you should be able to call the clear_edit_log command by itself by also passing in the `-g` flag to target just log entries for that graph

Reply:
Fixed in [4be40d4](https://github.com/archesproject/arches/pull/10837/commits/4be40d4d46a409d12faf10ea11f21438642572ef)